addon/templates/components/cf-field/info.hbs

```diff
@@ -1,6 +1,6 @@
 <button type="button" class="uk-icon-button" uk-icon="info"></button>
 
-<div uk-drop>
+<div uk-drop="pos: bottom-justify; boundary: .uk-form-controls; boundary-align: true">
```

Comment:
Is this a breaking change for other implementations?

Reply:
Nope. This just changes the visual appearance.